bears/haml/HAMLLintBear.py

```diff
@@ -0,0 +1,396 @@
+                        classes_before_ids: bool=True,
+                        consecutive_comments: bool=True,
+                        max_consecutive_comments: int=1,
+                        consecutive_silent_scripts: bool=True,
+                        max_consecutive_silent_scripts: int=2,
+                        empty_object_reference: bool=True,
+                        empty_script: bool=True,
+                        final_newline: bool=True,
+                        final_newline_present: bool=True,
+                        html_attributes: bool=True,
+                        id_names: bool=True,
+                        id_names_style: str='lisp_case',
+                        implicit_div: bool=True,
+                        indentation: bool=True,
+                        indentation_character: str='space',
+                        indentation_width: int=2,
+                        inline_styles: bool=True,
+                        instance_variables: bool=True,
+                        instance_variables_file_types: str='partials',
+                        instance_variables_matchers_all: str='.*',
+                        instance_variables_matchers_partials:
+                        str='\A_.*\.haml\z',
+                        leading_comment_space: bool=True,
+                        line_length: bool=True,
+                        max_line_length: int=80,
+                        multiline_pipe: bool=True,
+                        multiline_script: bool=True,
+                        object_reference_attributes: bool=True,
+                        repeat_id: bool=True,
+                        repeat_id_severity: str='error',
+                        rubo_cop: bool=True,
+                        rubo_cop_ignored_cops:
+                        typed_list(str)=DEFAULT_IGNORED_COPS,
+                        ruby_comments: bool=True,
+                        space_before_script: bool=True,
+                        space_inside_hash_attributes: bool=True,
+                        space_inside_hash_attributes_style: str='space',
+                        tag_name: bool=True,
+                        trailing_whitespace: bool=True,
+                        unnecessary_interpolation: bool=True,
+                        unnecessary_string_output: bool=True,
+                        view_length: bool=True,
+                        max_view_length: int=100):
+        """
+        :param alignment_tabs:
+            Use this rule to enable the linter to check if tabs are used within
+            a tag for alignment or not.
+        :param alt_text:
+            Use this rule to enable the linter
+            to check if alternate text is specified within img tags or not.
+            Using alt attributes is important to make the site more accessible.
+        :param class_attribute_with_static_value:
+            Use this rule to enable the linter to check if static class
```

Comment:
For all occurrences:
"~~Use this rule to enable the linter to check~~ ..." --> "Check ..."